### ISSUE TITLE
[CI] CoreValuesSourceTypeTests.testDatePrepareRoundingWithQuery failure

### ISSUE THREAD

**Issue by romseygeek** [elastic/elasticsearch#63659]
**Build scan**:
https://gradle-enterprise.elastic.co/s/sk3idodjsauxm/console-log?task=:server:internalClusterTest

**Repro line**:
./gradlew ':server:test' --tests "org.elasticsearch.search.aggregations.support.CoreValuesSourceTypeTests.testDatePrepareRoundingWithQuery" -Dtests.seed=953606DE8449AE7F -Dtests.security.manager=true -Dtests.locale=fr-BE -Dtests.timezone=Pacific/Yap -Druntime.java=11	

**Reproduces locally?**:
Yes

**Applicable branches**:
master, 7x

**Failure history**:
Just the once so far, but the test was added only a few days ago

**Failure excerpt**:
```
org.elasticsearch.search.aggregations.support.CoreValuesSourceTypeTests > testDatePrepareRoundingWithQuery FAILED
    Argument(s) are different! Wanted:
    rounding.prepare(
        1415,
        81897210183
    );
    -> at org.elasticsearch.search.aggregations.support.CoreValuesSourceTypeTests.lambda$testDatePrepareRoundingWithQuery$4(CoreValuesSourceTypeTests.java:79)
    Actual invocation has different arguments:
    rounding.prepare(
        -17514144000000,
        81897210183
    );
    -> at org.elasticsearch.search.aggregations.support.CoreValuesSourceType$6$1.lambda$roundingPreparer$0(CoreValuesSourceType.java:333)
        at __randomizedtesting.SeedInfo.seed([953606DE8449AE7F:84145B19A2F491A2]:0)
        at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
        at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
        at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
        at org.elasticsearch.search.aggregations.support.CoreValuesSourceTypeTests.lambda$testDatePrepareRoundingWithQuery$4(CoreValuesSourceTypeTests.java:79)
        at org.elasticsearch.index.mapper.MapperServiceTestCase.lambda$withAggregationContext$10(MapperServiceTestCase.java:357)
        at org.elasticsearch.index.mapper.MapperServiceTestCase.withLuceneIndex(MapperServiceTestCase.java:187)
        at org.elasticsearch.index.mapper.MapperServiceTestCase.withAggregationContext(MapperServiceTestCase.java:350)
        at org.elasticsearch.search.aggregations.support.CoreValuesSourceTypeTests.testDatePrepareRoundingWithQuery(CoreValuesSourceTypeTests.java:76)
```

**Comment by romseygeek** [comment]
@nik9000 this looks like it might be one of yours?

**Comment by nik9000** [comment]
I've filed #63680.

**Comment by nik9000** [comment]
Fixed by b0b2acc8c59c43308ea46b2b7a1b3944ae4e1b65.
